calculator/parser.py

```diff
@@ -0,0 +1,132 @@
+BINARY_OPERATORS = ["+", "-", "*", "/"]
+
+
+def binary_operator(num_1, num_2, op):
+    """
+    Evaluates and acts on different binary operators
+
+    Parameters
+    ----------
+    num_1 : integer
+        The first number to be operated on
+    
+    num_2 : integer
+        The second number to be operated on
+    
+    op : char
+        The operator that is going to be applied to the numbers
+
+    Returns
+    -------
+    integer : The final answer after after the nums are operated on
+    """
+
+    if op == "+":
+        return num_1 + num_2
+
+    elif op == "-":
+        return num_2 - num_1
+
+    elif op == "*":
+        return num_1 * num_2
+
+    elif op == "/":
+        return num_2 / num_1
+
+
+def push(e, stack):
+    """
+    Pushes the element (e) to the given stack
+
+    Parameters
+    ----------
+    e : element
+        The element to be pushed on top of the stack
+
+    stack : list
+        The stack that e is going to be pushed onto
+
+    Returns
+    -------
+    list : The stack that is now updated to have e pushed ontop of it
+    """
+    return stack.append(e)
+
+
+def pop(stack):
+    """
+    Retrieves the last element from the given stack and deletes it
+
+    Parameters
+    ----------
+    stack : list
+        The stack (in list form) to be operated on
+
+    Returns
+    -------
+    stack : The popped stack
+    """
+    return stack.pop()
```

Comment:
Same comment as above about the return.  Make sure you clean up the docstring to indicate what you're actually doing.